.github/filters/ci.yml

```diff
@@ -1,14 +1,14 @@
 ci-workflow: &ci-workflow .github/workflows/ci.yml
 
-rust-libparsec: &rust-libparsec oxidation/libparsec
+rust-libparsec: &rust-libparsec oxidation/libparsec/**
```

Comment:
What is this for ? it wasn't working properly before ?

Reply:
The `/**` was to watch the change recursively on the folder `oxidation/libparsec`